authui/src/authflowv2/components/input.css

```diff
@@ -0,0 +1,87 @@
+@layer components {
+  :root {
+    --input__ring-color: var(--color-grey-100);
+    --input__ring-color--focus: var(--color-primary-base);
+    --input__ring-color--error: var(--color-error);
+    --input__ring-width: var(--border-radius-regular);
+    --input__ring-width--focus: var(--border-radius-medium);
+    --input__ring-width--error: var(--border-radius-medium);
+    --input__border-radius: var(--border-radius-rounded-square);
+    --input__placeholder-color: var(--color-grey-200);
+    --input__text-color: var(--color-grey-700);
+    --input__font-family: var(--typography-body-large__font-family);
+    --input__font-size: var(--typography-body-large__font-size);
+    --input__line-height: var(--typography-body-large__line-height);
+    --input__letter-spacing: var(--typography-body-large__letter-spacing);
+    --input__font-weight: var(--typography-body-large__font-weight);
+    --input__y-padding: 1em;
+    --input__x-padding: 1.09em;
+
+    --input-error-message__text-color: var(--color-error);
+    --input-error-message__font-family: var(
+      --typography-body-medium__font-family
+    );
+    --input-error-message__font-size: var(--typography-body-medium__font-size);
+    --input-error-message__line-height: var(
+      --typography-body-medium__line-height
+    );
+    --input-error-message__letter-spacing: var(
+      --typography-body-medium__letter-spacing
+    );
+    --input-error-message__font-weight: var(
+      --typography-body-medium__font-weight
+    );
+  }
+}
+
+:root.dark {
+  /* TODO(davis): Dark mode styles */
+}
+
+.input {
+  padding: var(--input__y-padding) var(--input__x-padding);
+  border-radius: var(--input__border-radius);
+  border: var(--input__border-width) solid var(--input__border-color);
+  color: var(--input__text-color);
+  font-family: var(--input__font-family);
+  font-size: var(--input__font-size);
+  line-height: var(--input__line-height);
+  letter-spacing: var(--input__letter-spacing);
+  font-weight: var(--input__font-weight);
+  caret-color: var(--input__ring-color--focus);
+  @apply ring-inset	
+      ring-[length:var(--input\_\_ring-width)]
+      ring-[color:var(--input\_\_ring-color)];
+
+  &:focus {
+    @apply outline-none;
```

Comment:
Should this be a style exist in base stylesheet which apply to all input?

Reply:
```
  *:focus {
    @apply outline-none;
  }

```
Actually I thought it's set globally in `tailwind.css`. But don't know why it's not functioning on my own testing :derpthink: